internal/olm/operator/registry/index/registry_pod_test.go

```diff
@@ -91,6 +92,14 @@ var _ = Describe("RegistryPod", func() {
 				Expect(rp.DBPath).To(Equal("/database/index.db"))
 			})
 
+			It("should create a registry with custom database path", func() {
+				customDBPath := "/foo/bar/database.db"
```

Comment:
Do we allow a custom database? This seems like we're mixing 2 different items.

Reply:
> Do we allow a custom database?

I am not sure, in case, we don't support then we could remove `dbPath` parameter from `containerCommandFor` in the unit tests.

Reply:
At the same time, I don't see any logical reason for not supporting the custom database path.